examples/megacli.cpp

```diff
@@ -79,6 +79,9 @@ bool gVerboseMode = false;
 // new account signup e-mail address and name
 static string signupemail, signupname;
 
+// true by default, to register a new account v2 (false means account v1)
+bool signupV2;
```

Comment:
Maybe we want to initialize to true as default value.
```suggestion
bool signupV2 = true;
```